packages/app/gatsby-config.js

```diff
@@ -54,6 +54,28 @@ module.exports = {
         },
       },
     },
+    {
```

Comment:
see https://www.gatsbyjs.com/plugins/gatsby-plugin-react-i18next/#configure-the-plugin